src/python/src/grpc/_links/_transmission_test.py

```diff
@@ -94,7 +94,12 @@ def create_service_completion(self):
     return _intermediary_low.Code.OK, 'An exuberant test "details" message!'
 
   def assertMetadataEqual(self, original_metadata, transmitted_metadata):
-    self.assertSequenceEqual(original_metadata, transmitted_metadata)
+    # we need to filter out any additional metadata added in transmitted_metadata
+    # since implementations are allowed to add to what is sent (in any position)
+    keys, _ = zip(*original_metadata)
```

Comment:
Yeah you're likely right... the check here is slightly off.

I was trying for something roughly Pythonic that would give a nice failure message should invariants not hold (since I seem to remember a conversation some time ago that that was a worthy thing for Python tests).

Is there a nice idiomatic way of writing such a comparison? Or should I open-code it? Or do we add a comment that the test is slightly flawed but should do the right thing under reasonably expected conditions?